docs/tctl/admin/membership/list_db.md

```diff
@@ -0,0 +1,27 @@
+---
+id: list_db
+title: tctl admin membership list_db
+description: describing cluster membership items
```

Comment:
```suggestion
description: How to describe Cluster membership items
```
Bare minimum improvement - still lacking any real richness.

Reply:
Truly, I'm working with what I've been given, and it's not much